src/Symfony/Component/Form/Tests/Extension/Validator/Constraints/FormValidatorTest.php

```diff
@@ -149,11 +127,11 @@ public function testValidateIfChildWithValidConstraint()
         $this->assertNoViolation();
     }
 
-    public function testDontValidateIfParentWithoutCascadeValidation()
+    public function testDontValidateIfParentWithoutValidConstraint()
```

Comment:
Looks better! Could you name the three tests for the Valid constraint consistently?

* testValidateChildIfValidConstraint
* testDontValidateChildIfNoValidConstraint
* testValidateConstraintsOptionEvenIfNoValidConstraint